torch/shape/_ops.py

```diff
@@ -0,0 +1,931 @@
+R"("
```

Comment:
Maybe add a comment here describing that you want this while file to be a valid C++11 raw string literal, for #include'ing by C++ code.